crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -483,11 +483,11 @@ int crypto_sign_verify(ml_dsa_params *params,
   pre[0] = 0;
   pre[1] = ctxlen;
   OPENSSL_memcpy(pre + 2 , ctx, ctxlen);
-  return crypto_sign_verify_internal(params, sig, siglen, m, mlen, pre, 2 + ctxlen, pk);
+  return ml_dsa_verify_internal(params, sig, siglen, m, mlen, pre, 2 + ctxlen, pk);
 }
 
 /*************************************************
-* Name:        crypto_sign_open
+* Name:        ml_dsa_verify_message
```

Comment:
`sign_message` and `verify_message` variants are never used (we use sign/verify without the appended message on the end, so that signature sizes are constant)-- should we just remove them from the library?

Reply:
I usually take the position that if

1. a function is never used and
2. it is not exposed as part of our public API

we should remove it.